test/scss/_assert.scss

```diff
@@ -24,8 +24,8 @@
 
   @include test('False is not truthy') {
     @include assert-equal(
-      _true-is-truthy(false),
-      false);
```

Comment:
Can you run `gulp sasslint` to check for unnecessary white-space and other minor code-formatting cleanup? You can also use `npm run commit` to lint, test, and compile docs in one command.